client/stylesheets/puzzle.scss

```
@@ -633,6 +600,16 @@ table.puzzle-list {
   left: 0;
 }
 
+.popover.related-puzzle-popover {
+  max-width: none;
+  display: flex;
+  flex-direction: column;
+
+  .popover-body {
+    overflow: scroll;
```

Any reason why this is `scroll` rather than `auto`?
```suggestion
    overflow: auto;
```